aries-site/src/examples/templates/forms/ShippingExample.js

```diff
@@ -177,6 +177,33 @@ export const ShippingExample = () => {
     // Your submission logic here
   };
 
+  // provide order of formfields for validation
+  // to properly place focus on any errors or infos
+  const formFields = [
+    'firstName',
+    'lastName',
+    'address1',
+    'address2',
+    'city',
+    'state',
+    'zipcode',
+    'isBusiness',
+    'fullName-shipping',
+    'phone-shipping',
+    'email-shipping',
+  ];
+
+  const onValidate = validationResults => {
```

Comment:
I know why and what your doing but maybe add in a quick comment

Reply:
good idea.